cmd/crc/cmd/status.go

```diff
@@ -0,0 +1,73 @@
+package cmd
+
+import (
+	"fmt"
+	"github.com/docker/go-units"
+	"github.com/spf13/cobra"
+	"os"
+	"path/filepath"
+	"text/template"
+
+	"github.com/code-ready/crc/pkg/crc/constants"
+	"github.com/code-ready/crc/pkg/crc/errors"
+	"github.com/code-ready/crc/pkg/crc/machine"
+)
+
+func init() {
+	rootCmd.AddCommand(statusCmd)
+}
+
+var statusCmd = &cobra.Command{
+	Use:   "status",
+	Short: "Display status of the cluster",
+	Long:  "Show details about OpenShift cluster and crc vm",
+	Run: func(cmd *cobra.Command, args []string) {
+		runStatus()
+	},
+}
+
+var statusFormat = `CRC VM:     {{.CrcStatus}}
+OpenShift:  {{.OpenShiftStatus}}
+DiskUsage:  {{.DiskUsage}}
+CacheUsage: {{.CacheUsage}} (used by oc binary, Disk image)
```

Comment:
Is the output of the status command a good place to indicate where the crc cache directory is? 
"used by oc binary, Disk image" belongs in the documentation I think, as this needs more details for better understanding.

Reply:
Yes we can add that information here..